[2023-03-25T14:46:08.364+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: transform_fact_user.test.dbt_transform_postgres.unique_fact_user_id manual__2023-03-25T14:45:54.460895+00:00 [queued]>
[2023-03-25T14:46:08.380+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: transform_fact_user.test.dbt_transform_postgres.unique_fact_user_id manual__2023-03-25T14:45:54.460895+00:00 [queued]>
[2023-03-25T14:46:08.382+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-25T14:46:08.384+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-25T14:46:08.385+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-25T14:46:08.401+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): test.dbt_transform_postgres.unique_fact_user_id> on 2023-03-25 14:45:54.460895+00:00
[2023-03-25T14:46:08.409+0000] {standard_task_runner.py:55} INFO - Started process 120 to run task
[2023-03-25T14:46:08.417+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'transform_fact_user', 'test.dbt_transform_postgres.unique_fact_user_id', 'manual__2023-03-25T14:45:54.460895+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/transform_fact_user.py', '--cfg-path', '/tmp/tmpkvmcbqxo']
[2023-03-25T14:46:08.419+0000] {standard_task_runner.py:83} INFO - Job 98: Subtask test.dbt_transform_postgres.unique_fact_user_id
[2023-03-25T14:46:08.545+0000] {task_command.py:388} INFO - Running <TaskInstance: transform_fact_user.test.dbt_transform_postgres.unique_fact_user_id manual__2023-03-25T14:45:54.460895+00:00 [running]> on host 0aac1a38f752
[2023-03-25T14:46:08.620+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=transform_fact_user
AIRFLOW_CTX_TASK_ID=test.dbt_transform_postgres.unique_fact_user_id
AIRFLOW_CTX_EXECUTION_DATE=2023-03-25T14:45:54.460895+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-25T14:45:54.460895+00:00
[2023-03-25T14:46:08.622+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-25T14:46:08.624+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/dbt_transform_postgres && dbt run --models unique_fact_user_id']
[2023-03-25T14:46:08.641+0000] {subprocess.py:86} INFO - Output:
[2023-03-25T14:46:10.725+0000] {subprocess.py:93} INFO - [0m14:46:10  Running with dbt=1.4.5
[2023-03-25T14:46:11.312+0000] {subprocess.py:93} INFO - [0m14:46:11  Found 3 models, 7 tests, 0 snapshots, 0 analyses, 403 macros, 0 operations, 0 seed files, 0 sources, 0 exposures, 0 metrics
[2023-03-25T14:46:11.320+0000] {subprocess.py:93} INFO - [0m14:46:11
[2023-03-25T14:46:11.326+0000] {subprocess.py:93} INFO - [0m14:46:11  Nothing to do. Try checking your model configs and model specification args
[2023-03-25T14:46:13.080+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-03-25T14:46:13.114+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=transform_fact_user, task_id=test.dbt_transform_postgres.unique_fact_user_id, execution_date=20230325T144554, start_date=20230325T144608, end_date=20230325T144613
[2023-03-25T14:46:13.171+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-25T14:46:13.190+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
